pages/enterprise/setup.rst

```diff
@@ -4,46 +4,192 @@
 Setup
 *****
 
-Graylog Enterprise comes as a Graylog server plugin which need to be installed
-in addition to the Graylog open source setup.
+Graylog Enterprise comes as a single package that includes Graylog Server and the following plugins:
 
-Requirements
-============
+- Enterprise plugin
+- Integrations plugin
+- Enterprise Integrations plugin
 
```

Comment:
I would add something like:

All parts could be received also as a single package. 

Or similar with links to the integration plugin installation packe and the list with single package names

Reply:
Do you mean something like this? 

```
If you're already running an open source instance, you can `install these plugins separately <#update-graylog-instance-to-graylog-enterprise>`_ to convert it to an Enterprise instance. 
```

Reply:
I like the idea of having each plugin be a hyperlink to more information about it, but I'm not sure what page would contain more info on each plugin.